### ISSUE TITLE
delivered_amount reported by ripple-lib uses decimal XRP instead of drops like rippled and data.ripple.com

### ISSUE THREAD

**Issue by xrpscanner** [ripple/ripple-lib#1067]
For tx hash `A528A1161AEC5B21EC38AB7922EE5D8B13F1F00FBE87EDBA94FD5AD897B0016F`

1. rippled reports delivered amount as:
```
[xrpscan@xrpbox ~]$ /opt/ripple/bin/rippled tx A528A1161AEC5B21EC38AB7922EE5D8B13F1F00FBE87EDBA94FD5AD897B0016F | jq .result.meta.delivered_amount
"12600000"
```

2. data.ripple.com reports delivered amount as:
curl https://data.ripple.com/v2/transactions/A528A1161AEC5B21EC38AB7922EE5D8B13F1F00FBE87EDBA94FD5AD897B0016F | jq .transaction.meta.delivered_amount
```
[xrpscan@xrpbox ~]$ curl https://data.ripple.com/v2/transactions/A528A1161AEC5B21EC38AB7922EE5D8B13F1F00FBE87EDBA94FD5AD897B0016F | jq .transaction.meta.delivered_amount
"12600000"
```

3. Whereas, ripple-lib reports delivered amount as:
```
A528A1161AEC5B21EC38AB7922EE5D8B13F1F00FBE87EDBA94FD5AD897B0016F: {"currency":"XRP","value":"12.6"} 
```

This seems inconsistent with how the two main data source represent XRP amounts. Docs say that XRP amounts are represented as drops, and it would be great if _ripple-lib_ too follows what the docs say.

**Comment by intelliot** [comment]
Correct, the decision was made a long time ago to make the values in ripple-lib more readable by dividing by 1000000 and using "XRP" as the units instead of "drops". Our philosophy has changed, though. Drops is now the preferred unit, until the very end when the value is displayed in a UI -- at that point, it should be converted to XRP.

It's worth looking into the relevant methods to see what it would look like to change the values to drops. Feel free to open a PR.

**Comment by mvadari** [comment]
This issue should be resolved on 2.0, because no response modification is done.
